data/human/transport missions.txt

```diff
@@ -3334,93 +3334,93 @@ mission "Mafia Extortion"
 				`"I don't have time to talk."`
 					goto funeral
 			label nameless
-			`Your greeting triggers a grumpy reply about the value of anonymity in a place like this.`
+			`Your greeting gets a grumpy reply about the value of anonymity in a place like this.`
 			choice
 				`"Sounds like you are afraid of talking to me. Why is that?"`
 					goto mafia
-				`"What should I be afraid off that would happen to my ship?"`
+				`"What should I be afraid of happenning to my ship?"`
 					goto specifics
 				`"You sound like one of those mafia goons."`
 					goto mafia
 				`"Are you trying to extort me?"`
 					goto mafia
 			label specifics
-			`The man responds; "Well, a fleet of unfriendly ships in this system might just try to blow your ship to pieces."`
+			`The man responds, "Well, a fleet of unfriendly ships in this system might just blow your ship to pieces."`
 			choice
-				`"Ha ha, that is business as usual in this system."`
+				`"Ha, business as usual in this system."`
 					goto unconcerned
 				`"Are you trying to extort me?"`
 					goto mafia
 				`"And you have a way to prevent this from happening?"`
 					goto mafia
 				`"And you are the local goon that takes bribes to keep such ships docked?"`
 					goto mafia
-				`"That would be unfortunate, I often run jobs for some of the most dangerous people on this planet and they wouldn't be happy if that happens."`
+				`"That would be unfortunate, I run jobs for some of the most dangerous people on this planet and they wouldn't be happy if that happened."`
```

Comment:
I wonder if real tense agreement would be "...if that were to happen," ? but informally "if that happened" sounds better to me than "if that happens"